src/components/config/config.ts

```diff
@@ -6,6 +6,7 @@ const EXTENSION_CONFIG_KEY = "vs-kubernetes";
 const KUBECONFIG_PATH_KEY = "vs-kubernetes.kubeconfig";
 const KNOWN_KUBECONFIGS_KEY = "vs-kubernetes.knownKubeconfigs";
 const KUBECTL_VERSIONING_KEY = "vs-kubernetes.kubectlVersioning";
+const RESOURCES_TO_WATCH_KEY = "vs-kubernetes.resource-to-watch";
```

Comment:
Suggest `resources-to-watch` (plural).

Reply:
Also add to `contributions` in `package.json` and we should document it in the README though I can add that when I do the release.

Reply:
What do you mean by `contributions`? Is the `contribute` section? I leave the README to you, you will surely be better at explaining everything